engines/python/setup/djl_python/rolling_batch/rolling_batch_vllm_utils.py

```diff
@@ -75,6 +75,9 @@ def update_request_cache_with_output(request_cache: OrderedDict,
     if "prompt_tokens_details" not in request_cache[
             request_id] and request_output.prompt_logprobs:
```

Comment:
Curious, request_output should not have prompt_logprobs as well right?

Reply:
from what I can read, `request_output.prompt_logprobs` is set to `[float("nan")]` and seems like it's passing that check.